lib/insights_cloud/async/insights_scheduled_sync.rb

```diff
@@ -13,9 +13,16 @@ def plan
           return
         end
 
-        after_delay do
-          plan_full_sync
+        unless ForemanRhCloud.with_local_advisor_engine?
+          after_delay do
+            plan_full_sync
+          end
         end
+        plan_self
```

Comment:
you dont want _plan_self_ to run if it is not _with_local_advisor_engine?_

Reply:
Fixed to be in an if/else